<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>ScanSimDlgUI</class>
  <widget class="QDialog" name="ScanSimDlgUI" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>261</width>
        <height>233</height>
      </rect>
    </property>
    <property name="windowTitle" >
      <string>Scan Simulator</string>
    </property>
    <widget class="Q3GroupBox" name="groupBox4" >
      <property name="geometry" >
        <rect>
          <x>10</x>
          <y>10</y>
          <width>240</width>
          <height>170</height>
        </rect>
      </property>
      <property name="title" >
        <string>Parameters</string>
      </property>
      <widget class="QPushButton" name="goButton" >
        <property name="geometry" >
          <rect>
            <x>20</x>
            <y>130</y>
            <width>70</width>
            <height>31</height>
          </rect>
        </property>
        <property name="text" >
          <string>Simulate</string>
        </property>
      </widget>
      <widget class="QLineEdit" name="pyEdit" >
        <property name="geometry" >
          <rect>
            <x>90</x>
            <y>30</y>
            <width>51</width>
            <height>21</height>
          </rect>
        </property>
      </widget>
      <widget class="QLineEdit" name="pzEdit" >
        <property name="geometry" >
          <rect>
            <x>160</x>
            <y>30</y>
            <width>51</width>
            <height>21</height>
          </rect>
        </property>
      </widget>
      <widget class="QLineEdit" name="dyEdit" >
        <property name="geometry" >
          <rect>
            <x>90</x>
            <y>60</y>
            <width>51</width>
            <height>21</height>
          </rect>
        </property>
      </widget>
      <widget class="QLineEdit" name="dzEdit" >
        <property name="geometry" >
          <rect>
            <x>160</x>
            <y>60</y>
            <width>51</width>
            <height>21</height>
          </rect>
        </property>
      </widget>
      <widget class="QLineEdit" name="uzEdit" >
        <property name="geometry" >
          <rect>
            <x>160</x>
            <y>90</y>
            <width>51</width>
            <height>21</height>
          </rect>
        </property>
      </widget>
      <widget class="QLineEdit" name="uyEdit" >
        <property name="geometry" >
          <rect>
            <x>90</x>
            <y>90</y>
            <width>51</width>
            <height>21</height>
          </rect>
        </property>
      </widget>
      <widget class="QLineEdit" name="uxEdit" >
        <property name="geometry" >
          <rect>
            <x>20</x>
            <y>90</y>
            <width>51</width>
            <height>21</height>
          </rect>
        </property>
      </widget>
      <widget class="QLineEdit" name="dxEdit" >
        <property name="geometry" >
          <rect>
            <x>20</x>
            <y>60</y>
            <width>51</width>
            <height>21</height>
          </rect>
        </property>
      </widget>
      <widget class="QLineEdit" name="pxEdit" >
        <property name="geometry" >
          <rect>
            <x>20</x>
            <y>30</y>
            <width>51</width>
            <height>21</height>
          </rect>
        </property>
      </widget>
    </widget>
    <widget class="QPushButton" name="exitButton" >
      <property name="geometry" >
        <rect>
          <x>201</x>
          <y>190</y>
          <width>40</width>
          <height>31</height>
        </rect>
      </property>
      <property name="text" >
        <string>Exit</string>
      </property>
    </widget>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
  <connections>
    <connection>
      <sender>goButton</sender>
      <signal>clicked()</signal>
      <receiver>ScanSimDlgUI</receiver>
      <slot>goButton_clicked()</slot>
    </connection>
    <connection>
      <sender>exitButton</sender>
      <signal>clicked()</signal>
      <receiver>ScanSimDlgUI</receiver>
      <slot>exitButton_clicked()</slot>
    </connection>
  </connections>
</ui>
